### ISSUE TITLE
Io looks unrealistically yellow

### ISSUE THREAD

**Issue by 10110111** [Stellarium/stellarium#3001]
Currently, Stellarium's Io looks like this:

![stellarium-010](https://user-images.githubusercontent.com/6376882/213268952-b95650de-7e16-4952-b987-6b5b7749a2d3.png)

Compared to the rendering of Jupiter, it's much yellower,

![stellarium-009](https://user-images.githubusercontent.com/6376882/213266035-897afba1-e20f-4160-9395-74e316aa2b5e.png)

Apparently, the texture used was made (with maybe some post-processing) from the images made by Galileo (like [this one](https://en.wikipedia.org/wiki/File:Io_highest_resolution_true_color.jpg)), which were created from three narrow-band-filtered photos in NIR, green, and violet parts of the spectrum:

![](https://upload.wikimedia.org/wikipedia/commons/thumb/7/7b/Io_highest_resolution_true_color.jpg/240px-Io_highest_resolution_true_color.jpg)

Now, this narrow-band far-side-ends sampling of the spectrum cannot yield true colors. And this is corroborated by more color-friendly cameras, like those used for the [photos from Juno](https://en.wikipedia.org/wiki/File:Images_of_Io_from_the_Juno_spacecraft.png):

![](https://upload.wikimedia.org/wikipedia/commons/thumb/9/92/Images_of_Io_from_the_Juno_spacecraft.png/640px-Images_of_Io_from_the_Juno_spacecraft.png)

or [from Voyager 1](https://en.wikipedia.org/wiki/File:Io_VGR_South_polar_color_mosaic.jpg):

![](https://upload.wikimedia.org/wikipedia/commons/thumb/2/22/Io_VGR_South_polar_color_mosaic.jpg/186px-Io_VGR_South_polar_color_mosaic.jpg)

or especially [from Cassini](https://en.wikipedia.org/wiki/File:PIA02879_-_A_New_Year_for_Jupiter_and_Io.jpg), which shows that Io colors should be much closer to those of Jupiter, not as yellow as Stellarium displays:

![](https://upload.wikimedia.org/wikipedia/commons/thumb/6/65/PIA02879_-_A_New_Year_for_Jupiter_and_Io.jpg/500px-PIA02879_-_A_New_Year_for_Jupiter_and_Io.jpg)

**Comment by gzotti** [comment]
I guess we were using what's available. Can you find a better all-globe texture?

**Comment by 10110111** [comment]
> Can you find a better all-globe texture?

Didn't try, just documented what caught my eye when I was working on another thing. If no one finds the replacement after all attempts, at least the current texture should be manipulated to look more or less sensible.

**Comment by gzotti** [comment]
I'm not sure, did NASA point out how much sulfur was on Io with these yellowish images? Must admit, I rarely observe Io's surface...

**Comment by 10110111** [comment]
Do you think the amount of sulfur changed so radically between these photo sessions?

**Comment by gzotti** [comment]
No, it's indeed possible that the yellowish tint is not what the human eye would see, but an artifact of particular filters on Galileo's cameras. Just that this was the best texture at the time it was added.
